docs/api/v3.rst

```diff
@@ -1681,7 +1681,7 @@ Remote Organization listing
 
     The ``results`` in response is an array of remote organizations data.
 
-    :query string name: return remote organizations with matching name
+    :query string name__contains: return remote organizations with containing the name
```

Comment:
Oh, sorry. If we are already using `name=` to filter by `icontains`, we should use the same here. No need to change it to `__contains`. I'm sorry for the confusion here.